.npmrc

```diff
@@ -0,0 +1 @@
+save-exact=true
```

Comment:
This is what `package-lock.json` is for. If we have fixed versions for dependencies, not even patches would be updated on a fresh install of this module.

Reply:
Some projects published to npm do not follow semver. I got bitten by this recently, and the reply was basically that they didn't care about it

Reply:
Was it one of the dependencies of this project?

Reply:
It was a dependency of one of the @sinonjs projects, trying to find it now

Reply:
I would argue that only the one dependency, where the maintainers use npm without following the official npm semantics, should be a fixed version. Giving up on semver entirely just because someone else doesn't like it means we're actively helping to kill it.

Reply:
https://github.com/mishoo/UglifyJS2/commit/ef618332ea92db57e59f90f166035a0e7cf8a509#commitcomment-26906692

https://github.com/sinonjs/sinon-test/pull/85

Considering how the maintainers of that module talk to people in issues, I'd be happy to drop it as a dependency.

Reply:
Good to know. I also depend on them in some projects. Do we have an alternative module?

Reply:
I don't think that any of the `sinonjs` projects need to ship minified versions at all. I'd be happy to remove it from `sinon-test`, and then update all repositories to use default npm behaviour (which I am not a huge fan of, but it's the well beaten path)

Reply:
Makes sense.

Reply:
I'll update this PR to remove that commit, and set dependencies back to normal

Reply:
done